src/Widgets/ToolbarButton.vala

```diff
@@ -30,6 +30,7 @@ public class ENotes.ToolbarButton : Gtk.Button {
     private string second_half;
 
     construct {
+        new WordWrapper(); // used to enforce initialization of static members
         can_focus = false;
```

Comment:
I don't believe you need this, can you double check? 

(Checking the code from my phone, so can't test ATM)

Reply:
Yea, regexes weren't working anymore so I went and looked it up. If you don't do this, static members don't get initialized unless you call an instance method (and this new class doesn't have any). Got the tip from a gnome mailing list. If you know any less obscure way of getting it done let me know - it bothered me too. Appears to be a C limitation.

Reply:
Alright, in that case maybe it should be initiated in either the editor or in the main window class, since doing it there will cause a new instance of WordWrapper to be created on every toolbar button.

Or maybe you were right to use `class construct` there :thinking: (I've just never seen it before, but doesn't mean it's wrong :) )